app/classifier/index.cjsx

```diff
@@ -5,6 +5,7 @@ ClassificationSummary = require './classification-summary'
 {Link} = require 'react-router'
 tasks = require './tasks'
 drawingTools = require './drawing-tools'
+{session} = require '../lib/session'
```

Comment:
It requires the string from the exported object from '../lib/session' It should calculate the string on page load and remain the same until the page is reloaded.